scripts/packages/README

```diff
@@ -443,6 +443,12 @@ recv_addr
     turns on incremental state transfer. IST will use SSL if SSL is configured
     as described above. No default.
 
+recv_bind
```

Comment:
I have no problem changing the name of the parameter, I just want to state that I prefer the current naming. It shows the two variables are related as both seem to be used for the IST. And if the name is changed some other variables like uri_addr and uri_bind should also be changed.
What would you propose here?